test/fixtures/output/kafka/consumer/group/lag/summarize-onprem.golden

```diff
@@ -0,0 +1,11 @@
++-------------------+------------------+
+| Cluster           | cluster-1        |
+| Consumer Group    | consumer-group-1 |
+| Total Lag         |              110 |
+| Max Lag           |              100 |
+| Max Lag Consumer  | consumer-1       |
+| Max Lag Instance  | instance-1       |
+| Max Lag Client    | client-1         |
+| Max Lag Topic     | topic-1          |
+| Max Lag Partition |                1 |
```

Comment:
This `summarize` command differs from our usual list of subcommands. Would it make sense to move these fields to `kafka consumer group describe` since they're in terms of the consumer group (as far as I can tell)?

Reply:
It does make sense since all lag routes are based on a consumer group, but I think it would be more awkward to have some lag information in a higher level `kafka consumer group` command and the rest in `kafka consumer group lag`.